upgrade-1.x-to-2.x.sh

```diff
@@ -160,15 +186,19 @@ umount /var/lib/docker
 umount /resin-data
 
 # Save conf contents to /boot
-# XXX
+if [ -f "/mnt/conf/config.json" ]; then
+    # the boot partition might be mounted ro on some systems, remount
+    mount -o remount,rw "${boot_path}"
```

Comment:
When booting 1.8.0 device, I get `ro` mount, tried it a few times, let me check it a bit more.